frontend/packages/core/src/Input/checkbox.tsx

```diff
@@ -7,17 +9,85 @@ import {
   FormLabel,
   Grid,
 } from "@material-ui/core";
-import styled from "styled-components";
+import CheckIcon from "@material-ui/icons/Check";
 
-const FormControl = styled(MuiFormControl)`
-  width: 75%;
-`;
+const FormControl = styled(MuiFormControl)({
+  width: "75%",
+});
 
-const Checkbox = styled(MuiCheckbox)`
-  ${({ theme }) => `
-  color: ${theme.palette.text.primary};
-  `}
-`;
+const StyledCheckbox = styled(MuiCheckbox)({
+  color: "#6e7083",
+  height: "48px",
+  width: "48px",
+  borderRadius: "30px",
+  "&:hover": {
+    background: "#f5f6fd",
+  },
+  "&:active": {
+    background: "#d7daf6",
+  },
+  "&.Mui-checked": {
+    color: "#ffffff",
+    "&:hover": {
+      background: "#f5f6fd",
+    },
+    "&:active": {
+      background: "#d7daf6",
+    },
+    "&.Mui-disabled": {
+      color: "#e7e7ea",
+      ".MuiIconButton-label": {
+        color: "rgba(13, 16, 48, 0.38)",
+      },
+    },
+  },
+});
+
+const Icon = styled.div(
+  {
+    borderRadius: "2px",
+    boxSizing: "border-box",
+    height: "24px",
+    width: "24px",
+  },
+  props => ({
+    border: props["data-disabled"] ? "1px solid #e7e7ea" : "1px solid #6e7083",
+  })
```

Comment:
this could be styled from the parent without having to plumb props. IMO it's cleaner but we haven't written a style guide or made firm decisions here so it can be left as-is.